### ISSUE TITLE
Update translation to Danish, Norwegian, Finnish and Spanish (CLI)

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#302]
For Zonemaster-CLI there are two messages to be translated to Danish, Norwegian, Finnish and Spanish. You can provide the translations as comments to this issue.

There is a new option `--hints` to provide a private root zone for Zonemaster. **Provide translation for**:
```
Name of a root hints file to override the defaults.
```

There is a new error message when more than one domain name is provided on the command line. **Provide translation for**:
```
Only one domain can be given for testing. Did you forget to prepend an option with '--<OPTION>'?
```

Done:
- [ ] Danish
- [x] Norwegian
- [x] Finnish
- [x] Spanish

**Comment by samisalmensuo** [comment]
_Name of a root hints file to override the defaults._
'Root hints' tiedoston nimi jolla ohitetaan oletusasetukset.

_Only one domain can be given for testing. Did you forget to prepend an option with '--<OPTION>'?_
Vain yksi verkkotunnus voidaan antaa testattavaksi kerrallaan. Unohditko lisätä määrityksen eteen '--<OPTION>'?

**Comment by matsduf** [comment]
Thank you, @samisalmensuo! Your translation is included in PR #303. Please verify that is correct.

**Comment by repperille** [comment]
Name of a root hints file to override the defaults.
Navn på en 'root hints'-fil for å overstyre standardinnstillingene.

Only one domain can be given for testing. Did you forget to prepend an option with '--<OPTION>'?
Bare ett domene kan gis for testing. Glemte du å sette '--' foran en opsjon?

**Comment by matsduf** [comment]
@repperille, thanks. Your translation has been added to #303. Please verify there.

**Comment by huguei** [comment]
Name of a root hints file to override the defaults.
es: "Nombre del archivo 'root hints' para reemplazar los valores por defecto."

Only one domain can be given for testing. Did you forget to prepend an option with '--<OPTION>'?
es: "Solo puede indicarse 1 dominio para las pruebas. ¿Quizás incluyó una opción y olvidó el prefijo '--<OPCIÓN>'?

**Comment by matsduf** [comment]
@huguei, thank you. I have added your translation to #303. Please verify that it is correct.

**Comment by matsduf** [comment]
Resolved by #303, except da.po is not updated.

**Comment by haarbo** [comment]
Name of a root hints file to override the defaults.
Navnet på "root hints"-fil for at overstyre standardindstillingerne.

Only one domain can be given for testing. Did you forget to prepend an option with '--<OPTION>'?
Kun et domænenavn kan testes af gangen. Glemte du '--' foran en option '--<OPTION>'?
